ar_module2_8.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+var myDate = (function iffe() {
+  /* eslint-disable no-console */
+  /* eslint-disable no-unused-vars */
+  var privateDate = new Date(); // make this available to both get and set
```

Comment:
Ok! Thanks, I'll try it for the next assignment.

--Andy

On Tuesday, April 26, 2016, jamdon <notifications@github.com> wrote:

> In ar_module2_8.js
> <https://github.com/JSCRIPT300-Spring2016/Module_2/pull/10#discussion_r61109547>
> :
>
> > @@ -0,0 +1,122 @@
> > +'use strict';
> > +var myDate = (function iffe() {
> > +  /* eslint-disable no-console */
> > +  /* eslint-disable no-unused-vars */
> > +  var privateDate = new Date(); // make this available to both get and set
>
> It's not a deal-breaker, but instead of instantiating the date as soon as
> the module executes, set it to null by default, then only set a value when
> one of your module's methods are called. This isn't required, just a
> suggestion.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JSCRIPT300-Spring2016/Module_2/pull/10/files/b6bc115566c310a607a3b48c4b9325493597eb5e#r61109547>
>